x/epochs/keeper/epoch_test.go

```diff
@@ -25,9 +25,10 @@ func (suite *KeeperTestSuite) TestEpochLifeCycle() {
 	suite.Require().Equal(epochInfo, epochInfoSaved)
 
 	allEpochs := suite.App.EpochsKeeper.AllEpochInfos(ctx)
-	suite.Require().Len(allEpochs, 4)
+	suite.Require().Len(allEpochs, 5)
 	suite.Require().Equal(allEpochs[0].Identifier, "day") // alphabetical order
-	suite.Require().Equal(allEpochs[1].Identifier, "monthly")
-	suite.Require().Equal(allEpochs[2].Identifier, "stride_epoch")
-	suite.Require().Equal(allEpochs[3].Identifier, "week")
+	suite.Require().Equal(allEpochs[1].Identifier, "mint")
+	suite.Require().Equal(allEpochs[2].Identifier, "monthly")
+	suite.Require().Equal(allEpochs[3].Identifier, "stride_epoch")
+	suite.Require().Equal(allEpochs[4].Identifier, "week")
```

Comment:
should be reasonably easy to remove, not sure we should before mainnet. we do not use them afaik

Reply:
one reason it could be nice to keep them: it's possible we want to use these variables in the future. A bit more tricky to add them in with a software upgrade

Reply:
keeping in this PR